backend/src/main/java/gov/cdc/usds/simplereport/api/apiuser/UserMutationResolver.java

```diff
@@ -174,4 +174,10 @@ public User updateUserPrivilegesAndGroupAccess(
         username, orgExternalId, accessAllFacilities, facilityIdsToAssign, role);
     return new User(_us.getUserByLoginEmail(username));
   }
+
+  @AuthorizationConfiguration.RequireGlobalAdminUser
+  @MutationMapping
+  public ApiUser markUserRolesAndFacilitiesAsDeleted(@Argument String username) {
+    return _us.markUserRolesAndFacilitiesAsDeleted(username);
```

Comment:
only nit is can we name these `clearUserRolesandFacilities` or something like that? "mark as deleted" to me implies that there's an isDeleted flag we are flipping and that this is reversible, but we're just clearing them for good

Reply:
Will do! thank you, merethe!